### DISCUSSION TITLE
Opentracing / zipkin integration

### DISCUSSION THREAD

**Discussion by oliviermattei** [Akryum/vue-apollo#753] [Q&A]
Hello, 

It's should be great we can configure and integrate an opentracing tool like zipkin directly with vue apollo.

**Comment by Austio** [comment]
https://github.com/Akryum/vue-apollo/pull/718/files was thinking once this resolves that the natural next step would be to wrap this in tracing and then also wrap all the lower level promises in a way that could be traced.

**Comment by jcchavezs** [comment]
You can allow to plug a middleware. Let me know if you need asistence adding zipkin.

**Comment by oliviermattei** [comment]
I'm not yet comfortable with apollo client and are wrap vue, I'll look at

**Comment by Austio** [comment]
@jcchavezs Are you aware of any apollo client middleware that does this?  (couldn't find with my google-foo skills)

**Comment by jcchavezs** [comment]
Hi @Austio unfortunatelly not that I know.
